sycl/include/CL/sycl/detail/pi.h

```diff
@@ -86,16 +86,6 @@ typedef enum {
   PI_EVENT_QUEUED = CL_QUEUED
 } _pi_event_status;
 
-typedef enum {
-  PI_COMMAND_KERNEL_LAUNCH = CL_COMMAND_NDRANGE_KERNEL,
-  PI_COMMAND_MEMBUFFER_WRITE = CL_COMMAND_WRITE_BUFFER,
-  PI_COMMAND_MEMBUFFER_READ = CL_COMMAND_READ_BUFFER,
-  PI_COMMAND_USER = CL_COMMAND_USER,
-  PI_COMMAND_EVENTS_WAIT = CL_COMMAND_MARKER,
-  PI_COMMAND_MEMBUFFER_COPY = CL_COMMAND_COPY_BUFFER,
-  PI_COMMAND_MEMBUFFER_FILL = CL_COMMAND_FILL_BUFFER
-} _pi_command_type;
```

Comment:
These changes break the CUDA plugin, e.g. https://github.com/intel/llvm/blob/dc729a72b5bd815ffd7e118cd8d1992c5248ee4a/sycl/plugins/cuda/pi_cuda.cpp#L1634 *et al.*

Reply:
Those are renamed i.e., PI_COMMAND_MEMBUFFER_WRITE -> PI_COMMAND_TYPE_MEM_BUFFER_WRITE

Reply:
Yes, but the renaming was not propagated to the CUDA plugin.

Reply:
@fwyzard, https://github.com/intel/llvm/pull/1310 should fix that issue.

Reply:
Thanks, that worked.

Reply:
@fwyzard sorry about that. Why wasn't there any indication in the tests that it broke CUDA plugin? Was I needed to run some other tests by myself? @bader

Reply:
> @fwyzard sorry about that. Why wasn't there any indication in the tests that it broke CUDA plugin? Was I needed to run some other tests by myself?

+@pvchupin to reply

Reply:
There is no buildbot to check that at the moment, but we are working on adding it.
Meanwhile to check locally you can try this https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain-with-support-for-nvidia-cuda